### PR TITLE
JS Error body unhide should also support new boilerplate

### PR DESCRIPTION
**Pull request by sriramkrish85**
https://github.com/ampproject/amphtml/issues/1414

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -123,7 +123,10 @@ function compile(watch, shouldMinify) {
     // If there is a sync JS error during initial load,
     // at least try to unhide the body.
     wrapper: 'try{<%= contents %>}catch(e){setTimeout(function(){' +
-        'document.body.style.opacity=1},1000);throw e};'
+        'document.body.style.opacity=1;' +
```
Comment (cramforce) [inline gulpfile.js]: Please make a little var like:

`var s=document.body.style`; and then use it.

**Comment by cramforce** [comment]
Please check out the test error.

**Comment by sriramkrish85** [comment]
PTAL

File: gulpfile.js (hunk 1 above)
Comment (cramforce) [inline gulpfile.js]: Ping on this comment

Comment (sriramkrish85) [inline gulpfile.js]: Done

**Comment by sriramkrish85** [comment]
PTAL

**Comment by jridgewell** [comment]
LGTM

**Comment by cramforce** [comment]
LGTM

**Pull request merged**